sigopt/utils.py

```diff
@@ -16,3 +16,34 @@ def accept_sigopt_not_found():
     if ae.status_code != HTTPStatus.NOT_FOUND:
       raise
     handled.exception = ae
+
+# Refer: https://stackoverflow.com/questions/3041986/apt-command-line-interface-like-yes-no-input
+def query_yes_no(question, default="yes"):
+  """Ask a yes/no question via raw_input() and return their answer.
+
+  "question" is a string that is presented to the user.
+  "default" is the presumed answer if the user just hits <Enter>.
+      It must be "yes" (the default), "no" or None (meaning
+      an answer is required of the user).
+
+  The "answer" return value is True for "yes" or False for "no".
+  """
+  valid = {"yes": True, "y": True, "ye": True, "no": False, "n": False}
+  if default is None:
+    prompt = " [y/n] "
+  elif default == "yes":
```

Comment:
`valid[default.lower()] is True`, and you can catch `KeyError` to raise the value error